data/core/tips.json

```diff
@@ -173,5 +173,120 @@
     "type": "snippet",
     "category": "tip",
     "text": [ "Food from before cataclysm won't last forever.  Keep that in mind." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "You can make unusual things from usual stuff.  Go full MacGyver style if needed." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "There are fungi you want to eat, and those you want to burn.  You'll know the difference." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Having a backup weapon is thoughtful.  As well as having a backup plan." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Windows can't stop zombies but they can stop gas, bile, and smoke.  And your smell." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Wet and angry?  A dry towel will bring your smile back." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Like to gamble?  Use mutagen!  Be a winner today!" ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Famous last words:  Lets poke this electiric zombie with a metal stick." ]
```

Comment:
Typo: "electric".
Also this line is repeated again below.

Reply:
Also, should be **let's**